ToDoList/ToDoList.API.Tests/Utilities/EqualConstraintExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework.Constraints;
+
+using ToDoList.API.Models;
+
+namespace ToDoList.API.Tests.Utilities
+{
+    internal static class EqualConstraintExtensions
+    {
+        public static EqualConstraint UsingToDoComparer(this EqualConstraint constraint)
+            => constraint.Using(ToDoComparer.Instance.Value);
+
+        private class ToDoComparer : IEqualityComparer<ToDoModel>
+        {
+            public static Lazy<ToDoComparer> Instance 
```

Comment:
It is not necessary since this class is `private` itself, however, singletons are usually accompanied by a private constructor (so the control of instantiation aspect remains in the class's "hands")

Reply:
Thanks for hint!